gatorgrade/main.py

```diff
@@ -0,0 +1,18 @@
+import typer
+
+app = typer.Typer(add_completion=False)
+file = "gatorgrade.yml"
+
+
+@app.callback(invoke_without_command=True)
+def gatorgrade(file: str):
```

Comment:
This command setup would require the user to specify a file name (presumably for `gatorgrade.yml`) when running this command; however, I think it would be better if the file location was automatically found, or just assumed to be in a certain relative location.

Reply:
That makes a lot of sense, thank you! I removed file as a parameter for the gatorgrade command.